spec/controllers/admin/affiliate_boosted_contents_controller_spec.rb

```diff
@@ -1,12 +1,16 @@
-require 'spec_helper'
+# frozen_string_literal: true
 
 describe Admin::AffiliateBoostedContentsController do
   let(:config) { described_class.active_scaffold_config }
 
   describe 'export columns' do
     it 'contains the correct columns in the correct order' do
-      expect(config.export.columns.map(&:name)).to eq(%i(title url description
-        publish_start_on publish_end_on boosted_content_keywords match_keyword_values_only status))
+      expect(config.export.columns.map(&:itself)).to eq(
```

Comment:
Alternatively, we could call `to_a` instead of `map(&:itself)`:
```ruby
config.export.columns.to_a
=> [:title,
 :url,
 :description,
 :publish_start_on,
 :publish_end_on,
 :boosted_content_keywords,
 :match_keyword_values_only,
 :status]
 ```

Reply:
Changed.